ui/transaction_details_page.go

```diff
@@ -300,7 +300,10 @@ func (pg *transactionDetailsPage) txnInputs(gtx layout.Context) layout.Dimension
 	collapsibleBody := func(gtx C) D {
 		return pg.transactionInputsContainer.Layout(gtx, len(transaction.Txn.Inputs), func(gtx C, i int) D {
 			amount := dcrutil.Amount(transaction.Txn.Inputs[i].Amount).String()
-			acctName := fmt.Sprintf("(%s)", transaction.AccountName)
+			walletID := transaction.Txn.WalletID
+			accountNumber := transaction.Txn.Inputs[i].AccountNumber
+			accountName := common.wallet.GetAccountName(walletID, accountNumber)
+			acctName := fmt.Sprintf("(%s)", accountName)
 			walName := transaction.WalletName
 			hashAcct := transaction.Txn.Inputs[i].PreviousOutpoint
```

Comment:
This looks like a duplicate of what's in txOutputs. Those values look like they could be used directly in `txIORow method by just passing it the transaction as an argument.